### DISCUSSION TITLE
custom dialogs

### DISCUSSION THREAD

**Discussion by MannusEtten** [ookii-dialogs/ookii-dialogs-wpf#37] [Q&A]
it is also possible to create custom dialogs with this framework?

**Comment by augustoproiete** [comment]
@MannusEtten I'm not sure what you mean by custom dialogs. Could you give an example?

**Comment by MannusEtten** [comment]
something like this:

![afbeelding](https://user-images.githubusercontent.com/47241453/107888284-9f211180-6f0b-11eb-8f48-d6a243a353b7.png)

**Reply by augustoproiete** [reply]
@MannusEtten I'm not sure I understand your question. The dialog on your screenshot looks very specific to your particular app. It's the kind of dialog you'd create using WPF/XAML.
